ground/gcs/src/plugins/coreplugin/mainwindow.cpp

```diff
@@ -181,14 +181,28 @@ MainWindow::MainWindow() :
     registerDefaultContainers();
```

Comment:
From the commit comment it seems you are thinking the stylesheets are CSS or supposed to work the same as CSS, that's not the case. They work on the existing QWidget layout engine, so some of the CSS things would be "pants on head idiotic".